File: server/model/location.go
package model

import "time"

type Location struct {
	ID        int64      `json:"id"`
	Address   string     `json:"address"`
	Lattitude float64    `json:"lattitude"`
	Longitude float64    `json:"longitude"`
	Info      string     `json:"info"`
	CreatedAt time.Time  `json:"created_at"`
	UpdatedAt *time.Time `json:"updated_at"`
}

File: webapp/src/app/models/location.ts
export type Location = {
  latitude: number,
  longitude: number,
  id?: number,
  info: string,
  street: string,
  number: number,
  city: string,
  state: string,
  postal_code: string,
  formatted_address: string,
  created_at: Date,
  updated_at?: Date,
}
